### PR TITLE
Adapted code to unified API

### PR DESCRIPTION
**Pull request by emilwestergren**
@simonmika @sebastianbaginski peer review

* Breaks Vidhance API

### CODE REVIEW

File: source/draw/gpu/opengl/GraphicBufferYuv420Semiplanar.ooc [hunk 1]
```diff
@@ -54,7 +54,7 @@ GraphicBufferYuv420Semiplanar: class extends RasterYuv420Semiplanar {
 		result coordinateSystem = this coordinateSystem
 		result
 	}
-	kean_draw_gpu_android_graphicBufferYuv420Semiplanar_new: unmangled static func (backend: Pointer, nativeBuffer: Pointer, handle: Pointer, size: IntSize2D, stride: Int, format: GraphicBufferFormat, uvOffset: Int) -> This {
+	kean_draw_gpu_graphicBufferYuv420Semiplanar_new: unmangled static func (backend: Pointer, nativeBuffer: Pointer, handle: Pointer, size: IntSize2D, stride: Int, format: GraphicBufferFormat, uvOffset: Int) -> This {
```
Comment (sebastianbaginski) [inline source/draw/gpu/opengl/GraphicBufferYuv420Semiplanar.ooc]: I think `backend: Pointer, nativeBuffer: Pointer, handle: Pointer` can be simplified

Comment (emilwestergren) [inline source/draw/gpu/opengl/GraphicBufferYuv420Semiplanar.ooc]: You mean to `backend, nativeBuffer, handle: Pointer` ?

Comment (sebastianbaginski) [inline source/draw/gpu/opengl/GraphicBufferYuv420Semiplanar.ooc]: yes

**Comment by sebastianbaginski** [comment]
Review done.

**Comment by marcusnaslund** [comment]
Conflicts due to `IntSize`, sorry.

**Pull request merged**